Dockerfile

```diff
@@ -87,7 +92,7 @@ ENV PYTHONPATH=${PYTHONPATH}:$srcDir:$projectDir/tests:$projectDir/tools
 COPY Pipfile .
 COPY Pipfile.lock .
 RUN pip3 install pipenv==2022.4.8 && PIP_NO_CACHE_DIR=off PIP_NO_BINARY=shapely,pygeos pipenv install --system --deploy --ignore-pipfile
-RUN pip3 install shapely
+## RUN pip3 install shapely
```

Comment:
Has this been tested? If leaving out the shapely install works, let's just remove this line altogether instead of keeping the commented line in here without any notes as to why.

Reply:
can do